.github/workflows/release.yaml

```diff
@@ -22,37 +22,33 @@ jobs:
         env:
           LAMBDA_REMOTE_DOCKER: true
         run: yarn test
-      - id: checkIfWeShouldRelease
+      - id: tagExistsAndVersion
         name: Check if tags exist in NPM and GIT
         run: |
           set +e
-          export PACKAGE_NAME="$(node -p "require('./package').name")"
-          export PACKAGE_VERSION="$(node -p "require('./package').version")"
-          export PACKAGE_AND_VERSION="${PACKAGE_NAME}@${PACKAGE_VERSION}"
+          packageName="$(node -p "require('./package').name")"
+          packageVersion="$(node -p "require('./package').version")"
+          packageNameAndVersion="${packageName}@${packageVersion}"
 
-          git describe --exact-match --match "v${PACKAGE_VERSION}" HEAD^2 &>/dev/null
-          gitTagExists="$([ "$?" = "0" ] && BOOL="true" || BOOL="false"; echo $BOOL)"
-
-          npm view "${PACKAGE_AND_VERSION}" dist.tarball | grep "v${PACKAGE_VERSION}" > /dev/null
+          npm view "${packageNameAndVersion}" dist.tarball | grep "v${packageVersion}" > /dev/null
```

Comment:
I think this `grep "v..."` will always fail. At least in the case of `lambda-tools` and `prim`. neither response from `npm view` contains a v-prefixed version.

Reply:
So it would have tried to re-publish.  Good catch.